storagetransfer/storagetransfer_test.go

```diff
@@ -135,6 +208,13 @@ func TestMain(m *testing.M) {
 		log.Printf("couldn't delete Azure bucket: %v", err)
 	}
 
+	_, err = sqsClient.DeleteQueue(&sqs.DeleteQueueInput{
```

Comment:
This can probably use a defer and go closer to where it's initialized (as you do for pubsub).